__tests__/app.test.js

```diff
@@ -0,0 +1,43 @@
+process.env.NODE_ENV = "test";
```

Comment:
Don't really need this line. NODE_ENV will set to "test" automatically when test is running